feature-detects/webgl.js

```diff
@@ -7,13 +7,8 @@
   "polyfills": ["jebgl", "cwebgl", "iewebgl"]
 }
 !*/
-define(['Modernizr', 'createElement'], function(Modernizr, createElement) {
+define(['Modernizr'], function(Modernizr) {
   Modernizr.addTest('webgl', function() {
-    var canvas = createElement('canvas');
-    var supports = 'probablySupportsContext' in canvas ? 'probablySupportsContext' :  'supportsContext';
```

Comment:
is supportsConext also removed or why did you delete it too?

Reply:
That was the original name of the feature, so yes, it's also removed: https://github.com/whatwg/html/commit/2542a12cb25ee93534cbed1f31b5e1bc05fcdd0e.